index.html

```diff
@@ -11,6 +12,78 @@
 </head>
 
 <body>
+    <div class="container">
+
+        <header>
+          <nav>
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </nav>
+          <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+        
+      </header>
+      
+        <section class="cta">
+          <div class="cta-text">
+            <h1>Innovation<br> On<br> Demand</h1>
+            <button>Get Started</button>
+          </div>
+          <img src="img/header-img.png" alt="Image of a code snippet.">
```

Comment:
You might want to add a `font-size` setting for this `<h1>` element in your media queries - on mobile in particular, it still remains huge enough to cause a horizontal scrollbar to pop up.